examples/symbol/aggregate.ts

```diff
@@ -0,0 +1,87 @@
+    TransferTransactionBodyBuilder,
+    UnresolvedAddressDto,
+    UnresolvedMosaicBuilder,
+    UnresolvedMosaicIdDto,
+} from 'catbuffer-typescript';
+import { Converter, Key, SymbolAddress, SymbolDeadline, SymbolKeyPair, SymbolNetwork } from '../../src/core';
+
+const mosaicId = BigInt('0x091F837E059AE13C'); // Testnet mosaic id
+
+const aliceKeyPair = new SymbolKeyPair(Key.createFromHex('1C16D0C8804546EFB4B11584AFACB294DFABF0D41E8E345F1F74BB6CAD162066'));
+const bobKeyPair = new SymbolKeyPair(Key.createFromHex('C73DF84178896A5D63F91467E606A44B06AF3892DB4F55FAF9524DA902E0FF7E'));
+const bobAddressAlias = 'bob.address'; // Bob's address alias namespace name
+
+/**
+ * Build Symbol transaction using the factory and sign.
+ */
+const network = new SymbolNetwork('testnet', 0x98, '3B5E1FA6445653C971A50687E75E6D09FB30481055E3990C84B25E9222DC1155'); // Specify network
+const factory = network.createTransactionFactory();
+
+const aliceAddress = new SymbolAddress(network.createAddressFromPublicKey(aliceKeyPair.publicKey));
+const bobAddress = factory.fullNameToNamespaceId(bobAddressAlias); // Bob's unresolved address
+
+const aliceTransaction = new TransferTransactionBodyBuilder({
+    recipientAddress: new UnresolvedAddressDto(factory.toUnresolvedAddress(bobAddress)),
+    mosaics: [
+        new UnresolvedMosaicBuilder({
+            mosaicId: new UnresolvedMosaicIdDto(mosaicId),
+            amount: new AmountDto(BigInt(2000000)),
+        }),
+    ],
+    message: Converter.concat(Uint8Array.of(0x00), Converter.utf8ToUint8('This is Alice')),
+});
+
+const bobTransaction = new TransferTransactionBodyBuilder({
+    recipientAddress: new UnresolvedAddressDto(aliceAddress.getAddressBytes()),
+    mosaics: [
+        new UnresolvedMosaicBuilder({
+            mosaicId: new UnresolvedMosaicIdDto(mosaicId),
+            amount: new AmountDto(BigInt(2000000)),
+        }),
+    ],
+    message: Converter.concat(Uint8Array.of(0x00), Converter.utf8ToUint8('This is Bob')),
+});
+
+const bondedTransaction = factory.createAggregateBonded(SymbolDeadline.createFromAdjustedValue(100), BigInt(100), [
+    factory.toEmbedded(aliceTransaction, aliceKeyPair.publicKey),
+    factory.toEmbedded(bobTransaction, bobKeyPair.publicKey),
+]); // Build bonded transaction using the factory
+
+/**
+ * Sign transaction
+ */
+bondedTransaction.sign(aliceKeyPair);
+
+// Assuming Alice is the aggregate transaction's initiator. A fund lock transaction is required prior announcing the bonded transaction.
+const lockTransactionBuilder = new HashLockTransactionBodyBuilder({
+    mosaic: new UnresolvedMosaicBuilder({
+        mosaicId: new UnresolvedMosaicIdDto(mosaicId),
+        amount: new AmountDto(BigInt(2000000)),
+    }),
+    duration: new BlockDurationDto(BigInt(100)),
+    hash: new Hash256Dto(bondedTransaction.transactionHash),
+});
+const lock = factory.create(SymbolDeadline.createFromAdjustedValue(100), BigInt(100), lockTransactionBuilder);
```

Comment:
Is it possible to create relative deadlines (now+2 hours - epoch adjustment)? Static deadlines are not used when the network is runnin